conans/test/integration/package_info_test.py

```diff
@@ -235,3 +237,169 @@ def package_info(self):
+        """)
+
+        client = TestClient()
+        client.save({"conanfile_dep.py": dep, "conanfile_consumer.py": consumer,
+                     "galaxy/starlight/starlight.h": "",
+                     "lib/libstarlight": "",
+                     "galaxy/planet/planet.h": "",
+                     "lib/libplanet": "",
+                     "galaxy/iss/iss.h": "",
+                     "iss_libs/libiss": "",
+                     "bin/exelauncher": ""})
+        dep_ref = ConanFileReference("dep", "1.0", "us", "ch")
+        dep_pref = PackageReference(dep_ref, NO_SETTINGS_PACKAGE_ID)
+        client.run("create conanfile_dep.py dep/1.0@us/ch")
+        client.run("create conanfile_consumer.py consumer/1.0@us/ch")
+        package_folder = client.cache.package_layout(dep_ref).package(dep_pref)
+
+        expected_comp_starlight_include_paths = [os.path.join(package_folder, "galaxy", "starlight")]
+        expected_comp_planet_include_paths = [os.path.join(package_folder, "galaxy", "planet")]
+        expected_comp_launcher_include_paths = []
+        expected_comp_iss_include_paths = [os.path.join(package_folder, "galaxy", "iss")]
+        expected_comp_starlight_library_paths = [os.path.join(package_folder, "lib")]
+        expected_comp_launcher_library_paths = [os.path.join(package_folder, "lib")]
+        expected_comp_planet_library_paths = [os.path.join(package_folder, "lib")]
+        expected_comp_iss_library_paths = [os.path.join(package_folder, "iss_libs")]
+        expected_comp_starlight_binary_paths = [os.path.join(package_folder, "bin")]
+        expected_comp_launcher_binary_paths = [os.path.join(package_folder, "bin")]
+        expected_comp_planet_binary_paths = [os.path.join(package_folder, "bin")]
+        expected_comp_iss_binary_paths = [os.path.join(package_folder, "bin")]
+        expected_comp_starlight_libs = ["libstarlight"]
+        expected_comp_planet_libs = ["libplanet"]
+        expected_comp_launcher_libs = []
+        expected_comp_iss_libs = ["libiss"]
+        expected_comp_starlight_system_libs = []
+        expected_comp_planet_system_libs = []
+        expected_comp_launcher_system_libs = ["ground"]
+        expected_comp_iss_system_libs = ["solar", "magnetism"]
+
+        expected_global_include_paths = expected_comp_planet_include_paths + \
+            expected_comp_iss_include_paths + expected_comp_starlight_include_paths
+        expected_global_library_paths = expected_comp_starlight_library_paths + \
+            expected_comp_iss_library_paths
+        expected_global_binary_paths = expected_comp_starlight_binary_paths
+        expected_global_libs = expected_comp_planet_libs + expected_comp_iss_libs + \
+            expected_comp_starlight_libs + expected_comp_launcher_libs
+        expected_global_system_libs = expected_comp_iss_system_libs + \
+            expected_comp_launcher_system_libs
+
+        self.assertIn("GLOBAL Include paths: %s" % expected_global_include_paths, client.out)
```

Comment:
If we do not test attributes with paths, then we can check here the full line:

```
self.assertIn("GLOBAL Libs: a, b, d", client.out)
```

which, IMO, is easier to read